pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/apps.kruise.io/v1alpha1/CloneSet/testdata/desired-cloneset-nginx.yaml

```diff
@@ -5,33 +5,34 @@ metadata:
     app: sample
   name: sample
   namespace: test-cloneset
+  generation: 3
 spec:
   replicas: 4
   selector:
     matchLabels:
       app: sample
-      test: cloneset 
+      test: cloneset
   template:
     metadata:
       labels:
         app: sample
-        test: cloneset 
+        test: cloneset
     spec:
       volumes:
         - name: configmap
           configMap:
-            name: my-sample-config    
+            name: my-sample-config
       containers:
         - name: nginx
           image: nginx:alpine
-          env: 
-          - name: logData
-            valueFrom: 
-              configMapKeyRef:
-                name: mysql-config
-                key: log
-          - name: lowerData
-            valueFrom:
-              configMapKeyRef:
-                name: mysql-config
-                key: lower
+          env:
+            - name: logData
+              valueFrom:
+                configMapKeyRef:
+                  name: mysql-config
+                  key: log
+            - name: lowerData
+              valueFrom:
+                configMapKeyRef:
+                  name: mysql-config
+                  key: lowerr
```

Comment:
Note that the `.status.observedGeneration` of the federated resource has been `value 1` when the scenario with the current test data is formed, so that the `.status.observedGeneration` of the test result obtained by executing `karmadactl interpret -f customizations.yaml --operation aggregateStatus --desired-file testdata/desired-cloneset-nginx.yaml  --status-file testdata/status-file.yaml` will be `value 1` as well. Otherwise we get `value 0` by executing test-command.

Reply:
fixed @yike21